test/runtests.jl

```diff
@@ -286,8 +286,22 @@ const prob_pds_linmod_nonconservative_inplace = PDSProblem(linmodP!, linmodD!, [
             @test_throws "MPE can only be applied to production-destruction systems" solve(prob_ip,
                                                                                            MPE(),
                                                                                            dt = 0.25)
-            #TODO: Test that MPRK22 requires α ≥ 1/2 (not yet implemented)
-            #TODO: Test that MPRK22, MPRK43I, MPRK43II can only be applied to PDS (not yet implemented)
+            @test_throws "MPRK22 can only be applied to production-destruction systems" solve(prob_oop,
+                                                                                              MPRK22(1.0))
+            @test_throws "MPRK22 can only be applied to production-destruction systems" solve(prob_ip,
+                                                                                              MPRK22(1.0))
+            @test_throws "MPRK22 requires α ≥ 1/2." solve(prob_pds_linmod, MPRK22(0.25))
+            @test_throws "MPRK22 requires α ≥ 1/2." solve(prob_pds_linmod, MPRK22(0.25))
```

Comment:
I guess one of these tests should be for the in-place version and the other one for the out-of-place version?

Reply:
You're right!

Reply:
I'll add a function `get_constant_parameters` for `MPRK22`, just like for `MPRK43`. This way the constant parameters are the same for in-place and out-of-place implementations and it is sufficient to test the out-of-place version.